packages/gatsby-theme/src/components/Skip/SkipLink.js

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+
+const SkipLink = ({ children, id, ...props }) => {
+  return (
+    <Box
+      as='a'
+      tabindex='0'
+      href={`#${id}`}
```

Comment:
Maybe we could add an `aria-label` attribute for accessibility purposes, what do you think? It could be something like `aria-label={label || "Skip to main content"}`

Reply:
Sure! I'm going to add it, thanks @edvicaty 🙏